utils.h

```diff
@@ -226,10 +226,31 @@ inline std::string size_attr_str(const Token& t, const std::string& prefix = "")
     return count_attr_str(t, prefix);
 }
 
+inline std::string pcount(Decl* p, const std::string& prefix = "")
+{
+    if (p->attrs_ && (p->attrs_->string_ || p->attrs_->wstring_))
+        return prefix + p->name_ + "_len";
+    if (p->dims_ && !p->dims_->empty())
+        return "(size_t)1";
+    if (p->type_->tag_ == Foreign && p->attrs_ && p->attrs_->isary_)
```

Comment:
For a foreign type that is `isary`, can `size` or `count` be specified?

Reply:
We currently allow `size` or `count` be specified with a foreign `isary` type. However, neither of these attributes are effective. I think we should emit warnings to handle those cases.